frontend/src/components/IndividualModel/ModelDetailModal.js

```diff
@@ -153,6 +153,7 @@ class ModelDetailModal extends Component {
 
         return (
             <Modal
+                id="contained-modal"
```

Comment:
this should not be here? I noticed a few others like this... this was a change from an unrelated PR. Please make sure your code is clean before submitting.

When starting a new issue, make sure you're pulling from the `main` branch when it's clean and synced with the origin. Then, you know only the changes you're making are regarding the issue at hand.